internal/storage/migrations/0005_group_membership.sql

```diff
@@ -1,11 +1,11 @@
 -- +goose Up
 CREATE TABLE group_members (
-  group_id STRING NOT NULL REFERENCES groups(id),
-  subject_id STRING NOT NULL,
-
+  group_id VARCHAR NOT NULL REFERENCES groups(id),
+  subject_id VARCHAR NOT NULL,
   index group_memberships_subject_id_index (subject_id),
   primary key (group_id, subject_id)
 );
-
+CREATE INDEX IF NOT EXISTS group_memberships_subject_id_index ON group_members (subject_id);
```

Comment:
Is there a reason this is needed instead of relying on the index defined in the table?

Reply:
Meant to remove the index that's defined in the table. That explains why it already existed. 🤦🏼‍♀️  The one in the table doesn't work on postgres. Updating now